portfolio/src/main/webapp/index.html

```diff
@@ -5,14 +5,164 @@
     <title>My Portfolio</title>
```

Comment:
nit: might be nice to put your name in the title since the "welcome to helena's portfolio" bit only displays on one of the pictures.